TYPO3.Neos/Documentation/ExtendingNeos/CreatingAPlugin.rst

```diff
@@ -201,7 +201,7 @@ Linking to a Plugin
 
 Inside of your Plugin you can use the usual f:link and f:uri ViewHelpers from fluid to link to other ControllerActions::
```

Comment:
these should probably be updated as well.. ``f:link.action`` and ``f:uri.action`` instead